src/devices/Mcp23xxx/Mcp23xxx.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Iot.Device.Mcp23xxx
 {
-    public abstract class Mcp23xxx : IDisposable
+    public abstract class Mcp23xxx : IDisposable, IGpioController
```

Comment:
NIT: IDisposable is not required any longer since IGpioController is Disposable already

Reply:
Fixed.